.openzeppelin/addresses-sokol.json

```diff
@@ -95,18 +95,6 @@
     "proxy": "0xBbF3aad37298C997a485E270bD465ec3e6aCD2a7",
     "contractName": "VersionManager"
   },
-  "Market": {
-    "proxy": "0xbacB9F2F94D9957Ed39eBc10b20AF8Ab4e0C20e6",
-    "contractName": "Market"
-  },
-  "Inventory": {
-    "proxy": "0xAd02eD619f234c4eb252b8b3636522fA24F7F6A8",
-    "contractName": "Inventory"
-  },
-  "LevelRegistrar": {
-    "proxy": "0x330e167945eb4EC0CB3a28D63533789aba62610B",
-    "contractName": "LevelRegistrar"
-  },
```

Comment:
if we do this, that triggers a new contract deploy for these contracts, which means new addresses, which means you need to also update the subgraph to accommodate these new addresses (this is what I was concerned about in discord). Failure to do so will result in our subgraph no longer indexing these contracts.

Reply:
or wait--actually i remember now--these were the old commerce project's contracts. nevermind, these are not indexed by the subgraph

Reply:
@habdelra that makes sense, however I am removing this contract as it is no longer one of the contracts in the protocol - this artefact is a leftover